File: 0961-n-repeated-element-in-size-2n-array/0961-n-repeated-element-in-size-2n-array.cpp
class Solution {
public:
    int repeatedNTimes(vector<int>& nums) {
        unordered_set<int> set;
        for(int i=0;i<nums.size();i++){
            set.insert(nums[i]);
            if(set.size()!=i+1) return nums[i];
        }
        return 0;
    }
};

File: 0961-n-repeated-element-in-size-2n-array/0961-n-repeated-element-in-size-2n-array.java
class Solution {
    public int repeatedNTimes(int[] nums) {
        HashSet<Integer> set=new HashSet<>();
        for(int i=0;i<nums.length;i++){
            set.add(nums[i]);
            if(set.size()!=i+1) return nums[i];
        }
        return 1;
    }
}